src/NuGet.Clients/NuGet.Tools/xlf/Resources.cs.xlf

```diff
@@ -133,8 +133,8 @@
         <note />
       </trans-unit>
       <trans-unit id="Text_AuditSources_Description">
-        <source>Audit sources provide vulnerability data during restore without acting as package sources. If no audit sources are configured, NuGet Audit uses package sources. [Learn more about audit sources](https://learn.microsoft.com/nuget/reference/nuget-config-file#auditsources)</source>
-        <target state="new">Audit sources provide vulnerability data during restore without acting as package sources. If no audit sources are configured, NuGet Audit uses package sources. [Learn more about audit sources](https://learn.microsoft.com/nuget/reference/nuget-config-file#auditsources)</target>
+        <source>Audit sources provide vulnerability data during restore without acting as package sources. If no audit sources are configured, NuGet Audit uses package sources and suppresses warning NU1905. [Learn more about audit sources](https://learn.microsoft.com/nuget/reference/nuget-config-file#auditsources)</source>
```

Comment:
This is not matching dev at the moment:

https://github.com/NuGet/NuGet.Client/blob/af9df1d886dc55882dce56480b94ec564f19f1ae/src/NuGet.Clients/NuGet.Tools/xlf/Resources.cs.xlf#L136